modules/addons/ratelimit/main.tf

```diff
@@ -0,0 +1,37 @@
+provider "helm" {
+  kubernetes {
+    host                   = var.k8s_host
+    cluster_ca_certificate = base64decode(var.k8s_cluster_ca_certificate)
+    token                  = var.k8s_client_token
+  }
+}
+
+provider "kubectl" {
+  host                   = var.k8s_host
+  cluster_ca_certificate = base64decode(var.k8s_cluster_ca_certificate)
+  token                  = var.k8s_client_token
+  load_config_file       = false
+}
+
+resource "random_password" "redis" {
+  length = 16
+}
+
+resource "helm_release" "redis" {
```

Comment:
should we set the `image.tag` to `7.0.7` to basically enforce the version rotation? like we do for `cert-manager` and `elastic/eck`, i am not worried about `grafana`, `argocd`, `prometheus` - however the `eck`, `cert-manager` and `redis` may impact the product functionality directly. wdyt?

Reply:
I don't know what versions of redis this chart is compatible with, so I'd rather just keep it to the chart's default version.

Reply:
ack, it does push 7.0.7,  looked up over here: https://artifacthub.io/packages/helm/bitnami/redis

Reply:
In the Bitnami Redis chart, the version of the chart is disconnected from the Redis version. Keeping a fixed Redis image version here forces us to maintain the right version of the chart and make sure which ones are compatible. This is one of the reasons I initially wanted to use the custom deployment: depending on a non-official chart adds this extra complexity we have to deal with now. Honestly, I think it's not worth it.